packages/vm/tests/api/runCode.spec.ts

```diff
@@ -95,3 +95,22 @@ tape('VM.runCode: interpreter', (t) => {
     st.end()
   })
 })
+
+tape('VM.runCode: RunCodeOptions', (t) => {
+  t.test('should throw on negative value args', async (st) => {
+    const vm = new VM()
+
+    const runCodeArgs = {
+      value: new BN('-10'),
+    }
+
+    try {
+      await vm.runCode(runCodeArgs)
+      st.notOk('should not accept a negative call value')
```

Comment:
notOk is a boolean input, should use fail

```suggestion
      st.fail('should not accept a negative call value')
```